package.json

```diff
@@ -50,8 +50,7 @@
   },
   "peerDependencies": {
     "react": "^16.14.0 || ^17.0.2",
-    "react-dom": "^16.14.0 || ^17.0.2",
-    "react-datepicker": "4.6.0"
```

Comment:
Er was een reden dacht ik dat deze mee in de peerDeps staat, @ian-emsens-shd gaat dat het beste weten.

Reply:
Base styling zit mee in die package, om [duplicate dependencies](https://github.com/viaacode/hetarchief-client/blob/master/src/modules/shared/components/Datepicker/Datepicker.tsx#L5) te vermijden in `client` wordt die daar ingeladen.